### DISCUSSION TITLE
How many people are playing zone character 11

### DISCUSSION THREAD

**Discussion by flyhig** [share121/inter-knot#1048] [其他]
![Uploading 17229421736103aa8759b93ef45e7e1fbcd33c3abba439618acee605b058d1530b7c6486d805b.0.gif…]()

**Comment by jenemi** [comment]
i d like to,but unfortunately i have had not the lovely cool character 11,f**k hoyo,f**k hoyo pot!

**Comment by lllliufeng** [comment]
maka baka!!!

**Comment by qwq1949** [comment]
In realm of codes, a secret lies
Zone character 11, a wondrous guise
A mark of precision, a dash so fine
A symbol of order, in digital design

With precision's touch, it guides the way
A beacon of clarity, in digital day
A whisper of elegance, in code's dark night
Zone character 11, a shining light

In algorithms' dance, it takes its part
A subtle yet crucial, artistic start
A charm of beauty, in binary's might
Zone character 11, a love so bright

**Reply by SixRadium** [reply]
So brilliant！

**Reply by silverwolf510trillion** [reply]
You wanna a PHD?

**Comment by GanlinSama** [comment]
wow, a great poet😍

**Comment by Erdbronie** [comment]
You're right but I'll say 'Maka Pakka!'

**Comment by natihawi** [comment]
11 is the best!!!

**Comment by RushiShuo-Max** [comment]
Lu Xun once said:"the more difficult, the more you want to do. Reform is always not plain sailing."
